package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "shared-components",
-  "version": "0.4.10",
+  "version": "0.5.0",
```

Comment:
how are we determining the difference between patch and minor on this repo?

Reply:
brand new component added or new feature is like going from 4 -> 5
updates to existing is 4.1 -> 4.2 for example 
is how we have been doing it so far

Reply:
so if you update an 0.4.X feature in 0.5.X, you increment to 0.5.X+1?

Reply:
yeah!

Reply:
cool cool